### PR TITLE
Add JUnit-test RemoveCommandTest.java

### LINKED ISSUE #55: JUnit-Test RemoveCommandTest

### PR DESCRIPTION
**Pull request by SimJJ96**
Fixes #55

### CODE REVIEW

**Review by Leeyp — APPROVED** [review]
LGTM.

File: src/main/java/seedu/fridgefriend/command/RemoveCommand.java [hunk 1]
```diff
@@ -51,4 +48,17 @@ private void showMessage() {
                 + " food in the fridge.";
         Ui.printMessage(message);
     }
+
+    /**
+     * Return the actual message shown to user for JUnit-test.
+     *
+     * @return the actual message after remove food from fridge to the user
+     */
+    public String actualMessage() {
```
Comment (kwokyto) [inline src/main/java/seedu/fridgefriend/command/RemoveCommand.java]: Method names should be an action. I suggest "getMessagePrintedToUser"

**Review by kwokyto — APPROVED** [review]
Other than the naming of the method, LGTM

**Review by Vinci-Hu — APPROVED** [review]
LGTM

**Pull request merged**